### PR TITLE
Exclude status endpoint to prevent redirect in healthcheck verify

### PR DESCRIPTION
**Pull request by blacksam07**
#### Board:
* [Ticket #NUMBER_OF_THE_TICKET](link_goes_here)
---
#### Description:
Add `ssl_options` to exclude the status endpoint from ssl redirect and prevent issues with health-check verification 

based on https://api.rubyonrails.org/classes/ActionDispatch/SSL.html

#### Notes:
* Currently we have problem with the Load Balancer when we have the `force_ssl` enabled, with this setup we can omit the ssl redirect in the status endpoint
---
#### Tasks:
- [x] Add each element in this format
---
#### Risk:
*
---

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -54,6 +54,7 @@
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
+  config.ssl_options = { redirect: { exclude: ->(request) { request.path.include?('api/v1/status') } } }
```
Comment (JulianPasquale) [inline config/environments/production.rb:57]: This sounds like a very specific use case associated with the infrastructure of your project.
Why we need this now? We were able to deploy many apps with the existing config without any issue.

Comment (blacksam07) [reply config/environments/production.rb:57]: @JulianPasquale All internal apps that we have disable the `force_ssl` to skip this issue

**Comment by santib** [comment]
I don't get why is this needed. Please explain why someone would hit this endpoint without SSL and why can't it hit it with SSL

**Comment by blacksam07** [comment]
Currently all projects in RS that use a load balancer need to skip the `force_ssl` because breaks the load balancer health check, this issue is because the ELB health-check make the request over the HTTP, if the RAILS app have the force_ssl enable, redirect all HTTP request to the HTTPS and the ELB consider the application as unhealthy because get a 301 status.

another way is keep this setup in any section of the documentation, if we want to keep enable the `force_ssl` enable and have ELB with a health-check we should to include this setup

**Comment by blacksam07** [comment]
@santib after I looking all internal projects that no using `force_ssl` I was assume that we omit this issue and not look solutions about it, I saw this as a solution to have force_ssl enable with a proxy/ELB active and opened this PR 

1. About the infra setup, ELB can sent the HTTPS request but not works properly, continue failing and this setup means maintaining the certificates every x time
2. `assume_ssl` Look great, I didn't see this solution at the moment but we can check in the future

If you are ok, we can close PR

**Pull request closed without merging**